### PR TITLE
[WIP] Theme editor fixes

### LINKED ISSUE #888: UI strange visible defect in dashboard CoreThemes page
tested in Safari

In chrome everything ok, but there doesn't work back button from inner page http://localhost:3000/reaction/dashboard/ui/etWv4KmZ3AYv2Qr2F (content remains the same, but URL changes)

### PR DESCRIPTION
**Pull request by mikemurray**
Fixes #888 #1054 

Updated layout and issues with missing borders. #888 

Replaced color picker fields with and actual color picker.

Fixed issues caused my malformed CSS entered into fields. #1054

### CODE REVIEW

**Comment by aaronjudd** [comment]
```
npm WARN react-colorpickr@4.2.0 requires a peer of react@0.14.x but none was installed.
npm WARN react-colorpickr@4.2.0 requires a peer of react-dom@0.14.x but none was installed.
```

**Comment by aaronjudd** [comment]
i18n labels also appear to be broken in this branch.

**Comment by aaronjudd** [comment]
Error in startup:
```
23:55:08.187Z  INFO Reaction: Modified 2 documents while importing 2 to Tags
Exception in callback of async function: SyntaxError: Unexpected token }
    at Object.parse (native)
    at server/startup/i18n.js:25:32
    at runWithEnvironment (packages/meteor/dynamics_nodejs.js:110:1)
23:55:08.612Z  INFO Reaction: Initializing REACTION reaction-checkout
```

File: server/api/core/ui.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import autoprefixer from "autoprefixer";
 import cssAnnotation from "css-annotation";
 import { Shops, Themes } from "/lib/collections";
 import { Reaction } from "./core";
+import Logger from "/server/api";
```
Comment (jshimko) [inline server/api/core/ui.js]: This needs to be `import { Logger } from "/server/api";

**Comment by zenweasel** [comment]
We should reopen when we are ready to revisit.

**Pull request closed without merging**